DataFormats/FTLRecHit/interface/FTLUncalibratedRecHit.h

```diff
@@ -0,0 +1,67 @@
+#ifndef DATAFORMATS_FTLUNCALIBRATEDRECHIT
+#define DATAFORMATS_FTLUNCALIBRATEDRECHIT
+
+#include <vector>
+#include "DataFormats/DetId/interface/DetId.h"
+
+class FTLUncalibratedRecHit {
+
+  public:
+  
+  typedef DetId key_type;
+
+  enum Flags {
+          kGood=-1,                 // channel is good (mutually exclusive with other states)  setFlagBit(kGood) reset flags_ to zero 
+          kPoorReco,                // channel has been badly reconstructed (e.g. bad shape, bad chi2 etc.)
+          kSaturated,               // saturated channel
+          kOutOfTime               // channel out of time
+  };
+
+  FTLUncalibratedRecHit();
+  FTLUncalibratedRecHit(const DetId& detId, float ampl, float ped,
+                          float jit, float chi2, uint32_t flags = 0, uint32_t aux = 0);
+
+  ~FTLUncalibratedRecHit();
+  float amplitude() const { return amplitude_; }
+  float pedestal() const { return pedestal_; }
+  float jitter() const { return jitter_; }
+  float chi2() const { return chi2_; }
+  float outOfTimeEnergy() const { return OOTamplitude_; }
```

Comment:
can you point me to the specs documentation to see that all these energy, jitter, and chi2 values are all relevant.